templates/docker-compose.j2

```diff
@@ -47,7 +49,9 @@ services:
       US_HOST: "http://us:8083"
       US_MASTER_TOKEN: "fake-us-master-token"
       CONFIG_PATH: "/etc/backend-configs/api.yaml"
-{% if "DEV_MODE"|env(false) %}  
+{% if "DEV_MODE"|env(false) %}
+      SERVICE_CLIENT_ID: "bi"
+      SERVICE_CLIENT_SECRET: "4rKnCliKAmqfQUK0H1JVk6mvWhtGSXZfTdNrcUvAzlILNneA7n4PjddWRLYvTHVK"
```

Comment:
From my point of view it will be better to use variables

Reply:
any pros ? Tokens are saved and not gonna be changed every time